tools/getkeys/index.js

```diff
@@ -3,11 +3,12 @@
  * Licensed under the MIT License.
  */
 
-import fs from "fs";
-import os from "os";
-import path from "path";
-import util from "util";
-import child_process from "child_process";
+import child_process from "node:child_process";
+import fs from "node:fs";
+import os from "node:os";
+import path from "node:path";
+import util from "node:util";
```

Comment:
Now that we're on node 18+ We can probably get rid of the `node:util` dependency and just `import { appendFile } from "node:fs/promises";`